sentinel-extension/sentinel-datasource-redis/src/main/java/com/alibaba/csp/sentinel/datasource/redis/config/RedisConnectionConfig.java

```diff
@@ -0,0 +1,583 @@
+
+        private Builder() {
+        }
+
+
+        /**
+         * Set Redis host. Creates a new builder.
+         *
+         * @param host the host name
+         * @return New builder with Redis host/port.
+         */
+        public static RedisConnectionConfig.Builder redis(String host) {
+            return redis(host, DEFAULT_REDIS_PORT);
+        }
+
+        /**
+         * Set Redis host and port. Creates a new builder
+         *
+         * @param host the host name
+         * @param port the port
+         * @return New builder with Redis host/port.
+         */
+        public static RedisConnectionConfig.Builder redis(String host, int port) {
+
+            AssertUtil.notEmpty(host, "Host must not be empty");
+            AssertUtil.isTrue(isValidPort(port), String.format("Port out of range: %s", port));
+
+            Builder builder = RedisConnectionConfig.builder();
+            return builder.withHost(host).withPort(port);
+        }
+
+        /**
+         * Set Sentinel host. Creates a new builder.
+         *
+         * @param host the host name
+         * @return New builder with Sentinel host/port.
+         */
+        public static RedisConnectionConfig.Builder sentinel(String host) {
+
+            AssertUtil.notEmpty(host, "Host must not be empty");
+
+            RedisConnectionConfig.Builder builder = RedisConnectionConfig.builder();
+            return builder.withSentinel(host);
+        }
+
+        /**
+         * Set Sentinel host and port. Creates a new builder.
+         *
+         * @param host the host name
+         * @param port the port
+         * @return New builder with Sentinel host/port.
+         */
+        public static RedisConnectionConfig.Builder sentinel(String host, int port) {
+
+            AssertUtil.notEmpty(host, "Host must not be empty");
+            AssertUtil.isTrue(isValidPort(port), String.format("Port out of range: %s", port));
+
+            RedisConnectionConfig.Builder builder = RedisConnectionConfig.builder();
+            return builder.withSentinel(host, port);
+        }
+
+        /**
+         * Set Sentinel host and master id. Creates a new builder.
+         *
+         * @param host     the host name
+         * @param masterId sentinel master id
+         * @return New builder with Sentinel host/port.
+         */
+        public static RedisConnectionConfig.Builder sentinel(String host, String masterId) {
+            return sentinel(host, DEFAULT_SENTINEL_PORT, masterId);
+        }
+
+        /**
+         * Set Sentinel host, port and master id. Creates a new builder.
+         *
+         * @param host     the host name
+         * @param port     the port
+         * @param masterId sentinel master id
+         * @return New builder with Sentinel host/port.
+         */
+        public static RedisConnectionConfig.Builder sentinel(String host, int port, String masterId) {
```

Comment:
It's better to rename `sentinel` to `redisSentinel` as our project's name is also **Sentinel**, so the name of the method may be confusing.